tensorrt_llm/_torch/models/modeling_nemotron_nano.py

```
@@ -3038,96 +3025,88 @@ def _interleave_video_audio_embeddings(
-            self._encode_audio(audio_data_list) if audio_data_list else []
+        video_embeds, video_num_tokens = self.vision_encoder(
+            [self._vision_view(p, "video") for p in multimodal_params]
         )
+        embedded_audio = [p.multimodal_data["video"].get("audio") for p in multimodal_params]
+        audio_outputs = (
+            self._encode_audio([a for a in embedded_audio if a is not None])
+            if any(embedded_audio) and self.sound_encoder is not None else []
+        )
+        rows: List[torch.Tensor] = []
+        audio_cursor = 0
+        for i, (p, emb) in enumerate(zip(multimodal_params, video_embeds)):
```

Ruff (0.15.21)

[warning] `zip()` without an explicit `strict=` parameter

Add explicit value for parameter `strict=`

(B905)